src/mscorlib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -38,7 +38,6 @@ namespace System.Reflection.Emit
     using System.Runtime.Serialization;
```

Comment:
Nevermind, I guess.., I see countless other remnants of what looks CAS related. Just searching for "partial.*trust", grants, "full.*trust" and the like. It will probably take a long time to see the last of that and it's best to take it in manageable bites.